src/pages/Search/SearchAdvancedFiltersPage/SearchFiltersFromPage.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SearchFiltersParticipantsSelector from '@components/Search/SearchFiltersParticipantsSelector';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as SearchActions from '@userActions/Search';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function SearchFiltersStatusPage() {
```

Comment:
Nitpick: Since function in `To` filter is named `SearchFiltersToPage`, shouldn't this be named `SearchFiltersFromPage`?

Reply:
good catch 👍